internal/scheduler/legacyscheduler.go

```diff
@@ -758,100 +942,45 @@ func queueSelectionWeights(priorityFactorByQueue map[string]float64, aggregateRe
 	return rv
 }
 
+func max(a, b float64) float64 {
```

Comment:
think this exists in the standard library now

Reply:
Fixed